src/effects/presets/effectchainpresetmanager.cpp

```diff
@@ -365,10 +365,12 @@ void EffectChainPresetManager::setQuickEffectPresetOrder(
 
 void EffectChainPresetManager::savePreset(EffectChainPointer pChainSlot) {
     auto pPreset = EffectChainPresetPointer::create(pChainSlot.data());
-    savePreset(pPreset);
+    if (savePreset(pPreset)) {
+        pChainSlot->loadChainPreset(pPreset);
```

Comment:
I think it would be better to put this at the end of `savePreset(EffectChainPresetPointer)` instead of the returning booleans and doing things in different overloads, etc.

Reply:
Well, there it'll be difficult (impossible?) to figure `EffectChainPointer pChainSlot` required for reload.
Or we pass `pChainSlot` to `savePreset(EffectChainPresetPointer)`...

Though, looking at the call hierarchy, I don't see the benefit of splitting this into two functions anyway.

Reply:
Ah of course. Sorry I overlooked that. Then disregard my comment. 

I dislike the fact that we overload the `savePreset` method then (since the behavior and the return type now differ).

Reply:
Okay, so I could rename `savePreset(EffectChainPresetPointer)`, or merge both methods.
I prefer the latter.
What do you think?

Reply:
Good point.
renamed to `savePresetAndReload(EffectChainPointer)` and squashed.